src/components/kycInfo/PersonalDataAndDocumentsDetailed.tsx

```diff
@@ -0,0 +1,206 @@
+import { KycFieldMetadata } from '../../types'
+import { IdentificationDocumentType } from '@fiatconnect/fiatconnect-types'
+import { Buffer } from 'buffer'
+
+enum IdTypeFriendlyName {
+  IDC = 'Statue-issued ID',
+  PAS = 'Passport',
+  DL = `Driver's License`,
+}
+
+function idTypeFormatter(friendlyName: string): string {
+  switch (friendlyName) {
+    case IdTypeFriendlyName.IDC: {
+      return IdentificationDocumentType.IDC
+    }
+    case IdTypeFriendlyName.PAS: {
+      return IdentificationDocumentType.PAS
+    }
+    case IdTypeFriendlyName.DL: {
+      return IdentificationDocumentType.DL
+    }
+    default:
+      return ''
+  }
+}
+
+function idTypeReverseFormatter(idType: string | undefined): string {
+  switch (idType) {
+    case IdentificationDocumentType.IDC: {
+      return IdTypeFriendlyName.IDC
+    }
+    case IdentificationDocumentType.PAS: {
+      return IdTypeFriendlyName.PAS
+    }
+    case IdentificationDocumentType.DL: {
+      return IdTypeFriendlyName.DL
+    }
+    default:
+      return ''
+  }
+}
+const b64Encode = (input: string) => Buffer.from(input).toString('base64')
+
+export const personalDataAndDocumentsDetailedSchemaMetadata: Record<
```

Comment:
seems like this could extend personalDataAndDocuments, given the overlap between the two